lms/djangoapps/verify_student/api.py

```diff
@@ -33,3 +34,42 @@ def send_approval_email(attempt):
     else:
         email_context = {'user': attempt.user, 'expiration_datetime': expiration_datetime.strftime("%m/%d/%Y")}
         send_verification_approved_email(context=email_context)
+
+
+def create_verification_attempt(user, name, status, expiration_datetime=None):
+    """
+    Create a verification attempt.
+
+    Args:
+        user (User): the user (usually a learner) performing the verfication attempt
+        name (string): the name of the user
+        status (string): the initial status of the verification attempt
+        expiration_datetime (datetime, optional): When the verification attempt expires. Defaults to None.
+
+    Returns:
+        VerificationAttempt (VerificationAttempt): The created VerificationAttempt instance
+    """
+    verification_attempt = VerificationAttempt.objects.create(
+        user=user,
+        name=name,
+        status=status,
+        expiration_datetime=expiration_datetime,
+    )
+
+    return verification_attempt.id
+
+
+def update_verification_status(attempt_id, status):
+    """
+    Update the VerificationAttempt status.
```

Comment:
```suggestion
    Update the status of a verification attempt.
```

Nit: to be parallel with `create_verification_attempt`.